lib/ed25519.js

```diff
@@ -0,0 +1,145 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  sign,
+  verify,
+  createPrivateKey,
+  createPublicKey,
+  randomBytes
+} from 'crypto';
+
+const {promisify} = require('util');
+
+const randomBytesAsync = promisify(randomBytes);
```

Comment:
Don't use `require` (use ESM import):

```suggestion
import {
  sign,
  verify,
  createPrivateKey,
  createPublicKey,
  randomBytes
} from 'crypto';
import promisify from 'util';

const randomBytesAsync = promisify(randomBytes);
```

Reply:
resolved this here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/07ee808ebc5569c235809a7d7645b9b18a26fef4

You did need

```js
import {promisify} from 'util'
```